client-src/elements/chromedash-enterprise-release-notes-page.ts

```diff
@@ -205,7 +227,7 @@ export class ChromedashEnterpriseReleaseNotesPage extends LitElement {
       ([milestone, platforms]) => ({
         stage_type: STAGE_ENT_ROLLOUT,
         rollout_milestone: Number(milestone),
-        rollout_platforms: Array.from(platforms),
+        rollout_platforms: Array.from(platforms as Set<string>),
```

Comment:
Adjust the `new Set()` call above to include a type, rather than casting down here.